examples/with-pages-router/.env.example

```
@@ -0,0 +1,8 @@
+API_URL=https://api.example.com
+APP_NAME=Pages Router Example
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)